examples/OpenML_Tutorial.ipynb

```diff
@@ -216,18 +251,18 @@
       "text/plain": [
        "    did             name  NumberOfInstances  NumberOfFeatures  NumberOfClasses\n",
        "1     1           anneal              898.0              39.0              6.0\n",
-       "2     2           anneal              898.0              39.0              6.0\n",
+       "2     2           anneal              898.0              39.0              5.0\n",
```

Comment:
How can the number of classes of a dataset change? And why only for dataset 2 and not 1, given that they're almost exactly the same thing?